configs/config.toml.SAMPLE

```diff
@@ -54,7 +54,17 @@
 [Storage]
   DataDir = "/my/secure/location"
 
+# Uncomment to enable the metrics module which collects and exposes information
+# useful for external monitoring tools. The port on which the `/metrics`
+# endpoint will be available and the frequency with which the metrics will be
+# collected can be customized using the below parameters.
```

Comment:
Let's list what kind of metrics are available as well as what types of values those are to explain the difference between `Metrics` and `Diagnostics` modules. My proposition:

```
# Uncomment to enable the metrics module which collects and exposes information
# useful for external monitoring tools usually operating on time series data. All values
# exposed by metrics module are quantifiable or countable. 
#
# The following metrics are available:
# - connected peers count
# - connected bootstraps count
# - eth client connectivity status
#
# The port on which the `/metrics` endpoint will be available and the frequency with
# which the metrics will be collected can be customized using the below parameters.
```